NumLib/DOF/DOFTableUtil.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef NUMLIB_DOF_DOFTABLEUTIL_H
+#define NUMLIB_DOF_DOFTABLEUTIL_H
+
+#include "NumLib/DOF/LocalToGlobalIndexMap.h"
+
+namespace NumLib
+{
+//! Returns nodal indices for the item identified by \c id from the given
+//!  \c dof_table.
+std::vector<GlobalIndexType> getIndices(
+    std::size_t const id, NumLib::LocalToGlobalIndexMap const& dof_table);
```

Comment:
All correct. Maybe better to call it `mesh_item_id` for now, like it is in [LocalToGlobalIndexMap](https://github.com/ufz/ogs/blob/master/NumLib/DOF/LocalToGlobalIndexMap.h#L78) for some functions.

If we decide to remove the possibility of non-element-wise (*i.e.* node-wise, or edge-wise) assembly, then we can restrict the name to element_id (and maybe save one or two lines at places where the elements are available.)